File: test_scripts/fib.py
k = float(700000)

def fibo(n):
    i = 3

    fib, last_fib = float(1), float(1)
    while i <= n:
        last_fib, fib = fib, fib + last_fib
        i = i + 1

    return fib

try:
    print(str(k) + "th fib is", fibo(k))
except:
    pass

File: test_scripts/fib.lua
local k = 700000

local function fibo(n)
    local i = 3

    local fib, last_fib = 1, 1
    while i <= n do
        last_fib, fib = fib, fib + last_fib
        i = i + 1
    end
    return fib
end

print(tostring(k) .. "th fib is", fibo(k))
